vast.conf

```
@@ -1,116 +1,380 @@
+; This is an example configuration file for VAST, striving to show all available
+; options. Lines starting with a semicolon are commented out. Options in angle
+; brackets have their default value determined at runtime.
+
+; Options that apply to VAST.
 system {
-;; The host and port to listen at and connect to.
-; endpoint = "localhost:42000"
+  ; The host and port to listen at and connect to.
+  ;endpoint = "localhost:42000"
+
+  ; The file system path used for persistent state.
+  ;db-directory = "vast.db/"
+
+  ; The file system path used for log files.
+  ;log-file = <server.log in db-directory>
+
+  ; Number of events to be packaged in a table slice (this is a target value that
+  ; can be underrun if the source has a low rate).
+  ;table-slice-size = 100
+
+  ; The table slice type (default|arrow).
+  ;table-slice-type = "default"
+
+  ; The size of an index shard.
+  ;max-partition-size = 1000000
+
+  ; The unique ID of this node.
+  ;node-id = "node"
```

```suggestion
  ;node-id = "vast"
```

Does this make sense?

---

The default value is node. This PR merely documents the default values.